<launch>
	<arg name="grasp_demonstrations_path" value="$(find grasp_template_planning)/data/grasp_demonstrations_data/" /> 
	<arg name="robot" default="armrobot"/>
	
  <include file="$(find tabletop_segmenter)/launch/tabletop_segmenter.launch">
	<arg name="tabletop_segmentation_points_in" value="/camera/depth/points" />
    <arg name="tabletop_segmentation_depth_in" value="/camera/depth/image_raw" />
    <arg name="tabletop_segmentation_rgb_in" value="/camera/rgb/image_color" />
    <arg name="tabletop_segmentation_cam_info" value="/camera/depth/camera_info" />
	<arg name="tabletop_segmentation_srv" value="template_tabletop_segmentation"/>
	<arg name="tabletop_segmentation_markers" value="template_tabletop_segmentation_markers"/>
  </include>
	
	<node pkg="grasp_template_planning" type="user_demonstration_recorder" name="user_demonstration_recorder" args="$(arg grasp_demonstrations_path)$(arg filename)" output="screen">
		<rosparam file="$(find grasp_template_planning)/config/template_config_$(arg robot).yaml" command="load"/>
	</node>
</launch>